docs/pipelineruns.md

```diff
@@ -479,6 +479,24 @@ If used with this `Pipeline`,  `build-task` will use the task specific `PodTempl
 `PipelineTaskRunSpec` may also contain `StepOverrides` and `SidecarOverrides`; see
 [Overriding `Task` `Steps` and `Sidecars`](./taskruns.md#overriding-task-steps-and-sidecars) for more information.
 
+An optional `Metadata` field can be added for each task to have the flexibility to add annotations and labels required in a specific running context.
```

Comment:
Let's be clear that only labels and annotations are supported, not the other fields in k8s metadata.

Reply:
let me add a specification for this term.